compiler/rustc_middle/src/middle/privacy.rs

```diff
@@ -27,8 +26,8 @@ pub enum AccessLevel {
 }
 
 /// Holds a map of accessibility levels for reachable HIR nodes.
-#[derive(Clone)]
-pub struct AccessLevels<Id = HirId> {
+#[derive(Debug)]
+pub struct AccessLevels<Id = LocalDefId> {
```

Comment:
Is this used somewhere with another `Id`?

Reply:
Yes, rustdoc uses `AccessLevels<DefId>` and actually adds foreign ids to the table as public or exported, I didn't investigate why exactly.

Reply:
IIRC it's because it needs to do its own privacy pass because rustc's is incorrect (#64762). It may be possible to remove when that's fixed, but don't quote me on that.